### PR TITLE
Enoch/ac 34 people api doc timezone

### PR DESCRIPTION
**Pull request by ExtraGravity**

### CODE REVIEW

File: src/routers/people.ts [hunk 1]
```diff
@@ -49,10 +49,10 @@ router.get('/', async (req, res) => {
  * @apiName GetPeopleInCategory
  * @apiGroup People
  * @apiVersion  0.0.1
- * @apiDescription Returns all people of this category and all their associated attributes and values.
- *              Parameters have no effect on this request, but the URL
- *              specifies what kind of people are given.
- * @apiName GetPeopleCategory
+ * @apiDescription Returns all people of this category and all their
+ * 				   associated attributes and values.
```
Comment (beaesguerra) [inline src/routers/people.ts]: weird tabbings?

File: src/routers/people.ts [hunk 2]
```diff
@@ -62,15 +62,15 @@ router.get('/', async (req, res) => {
   [
     {
       "personUuid": "1e167b81-d816-497b-8c0c-36f4d6b2fd33",
+      "lastModified": "2018-01-23 04:05:06Z.123"
```
Comment (beaesguerra) [inline src/routers/people.ts]: Z is after ms?

File: src/routers/people.ts [hunk 3]
```diff
@@ -62,15 +62,15 @@ router.get('/', async (req, res) => {
...
       "firstName": "Zachariah",
-      "lastModified": "2018-01-23 04:05:06"
       "paymentFrequency": "weekly",
       "notes": "Brother of Moses",
     },
     {
       "personUuid": "4b3f23a3-04c4-468f-bdf5-f189a34d9f69",
       "firstName": "Zachariah",
-      "lastModified": "2018-01-23 04:05:06"
+      "lastModified": "2018-01-23 04:05:06Z.123"
```
Comment (beaesguerra) [inline src/routers/people.ts]: z is after ms

File: src/routers/people.ts [hunk 4]
```diff
@@ -88,7 +88,6 @@ router.get('/:category', async (req, res) => {
  * @apiDescription Creates a new person in the specified category.
  *              Returns the UUID created for that person.
  *              Requires all associated attributes given in /peopleCategories
- * @apiName CreatePerson
```
Comment (beaesguerra) [inline src/routers/people.ts]: why was this removed?

File: src/routers/people.ts [hunk 5]
```diff
@@ -121,7 +119,7 @@ router.post('/:category', async (req, res) => {
  * @apiSuccessExample /people/farmers/1e167b81-d816-497b-8c0c-36f4d6b2fd33 Success-Response:
    {
      "personUuid": "1e167b81-d816-497b-8c0c-36f4d6b2fd33",
-     "lastModified": "2018-01-23 04:05:06"
+     "lastModified": "2018-01-23 04:05:06Z.123"
```
Comment (beaesguerra) [inline src/routers/people.ts]: z is after ms

File: src/routers/transactions/products.ts [hunk 6]
```diff
@@ -8,14 +8,96 @@ import { StatusCode } from '@/models/statusCodes';
...
+  ]
+ */
 router.get('/:type', async (req, res) => {
   res.status(StatusCode.OK).send('Successfully retrieved all product <type> transactions');
 }, authorized(UserType.ADMIN));
 
+/**
+ * @api {post} /transactions/products/<type>
+ * @apiDescription Returns all product transactions of a specified type.
+ *              Required fields are:
```
Comment (beaesguerra) [inline src/routers/transactions/products.ts]: use apiparam please!

File: src/routers/transactions/products.ts [hunk 7]
```diff
@@ -8,14 +8,96 @@ import { StatusCode } from '@/models/statusCodes';
...
+ *                - amountOfProduct
+ *                - costPerUnit
+ *                - currency
+ *                Along with any associated attributes.
+ *                Associated attirubtes can be checked via /products
+ *                The product transaction UUID is returned on success.
+ *
+ * @apiName CreateNewProductTransaction
+ *
+ * @apiError (400) The required fields are invalid or missing, ...
```
Comment (beaesguerra) [inline src/routers/transactions/products.ts]: need the "badrequest" part?

File: src/routers/transactions/products.ts [hunk 8]
```diff
@@ -8,14 +8,96 @@ import { StatusCode } from '@/models/statusCodes';
...
+ */
 router.post('/:type', async (req, res) => {
   res.status(StatusCode.CREATED).send('Successfully created new product <type> transaction');
 }, authorized(UserType.ADMIN));
 
+/**
+ * @api {put} /transactions/products/<type>/<uuid>
+ * @apiDescription Updates a product transaction.
+ *              Available fields are:
+ *                - datetime
```
Comment (beaesguerra) [inline src/routers/transactions/products.ts]: use apiparam please!

File: src/routers/transactions/products.ts [hunk 9]
```diff
@@ -8,14 +8,96 @@ import { StatusCode } from '@/models/statusCodes';
...
+ *                - productTypeId
+ *                - amountOfProduct
+ *                - costPerUnit
+ *                - currency
+ *                Along with any associated attributes.
+ *                Associated attirubtes can be checked via /products
+ *
+ * @apiName UpdateProductTransaction
+ *
+ * @apiError (400) The fields are invalid, ...
```
Comment (beaesguerra) [inline src/routers/transactions/products.ts]: need "badrequest"

File: src/routers/transactions/products.ts [hunk 10]
```diff
@@ -8,14 +8,96 @@ import { StatusCode } from '@/models/statusCodes';
...
+ * @apiError (401) Unauthorized - Must be admin
+ * @apiError (404) The product transaction has not been found.
```
Comment (beaesguerra) [inline src/routers/transactions/products.ts]: need "notfound"

File: src/routers/transactions/products.ts [hunk 11]
```diff
@@ -8,14 +8,96 @@ import { StatusCode } from '@/models/statusCodes';
...
+ *
+ * @apiSuccess (200) {String} Successfully updated <type> transaction
```
Comment (beaesguerra) [inline src/routers/transactions/products.ts]: need "success"

**Review by beaesguerra — CHANGES_REQUESTED** [review]

File: src/routers/people.ts (hunk 4 above)
Comment (ExtraGravity) [reply src/routers/people.ts]: It's on the lines above, duplicated.

**Review by beaesguerra — APPROVED** [review]
